packages/discord.js/src/util/Transformers.js

```diff
@@ -33,4 +33,14 @@ function _transformAPIAutoModerationAction(autoModerationAction) {
   };
 }
 
-module.exports = { toSnakeCase, _transformAPIAutoModerationAction };
+/**
+ * Transforms an API authorizing integration owners map to keyed by enum.
+ * @param {APIAuthorizingIntegrationOwnersMap} authorizingIntegrationOwners The data to transform
+ * @returns {AuthorizingIntegrationOwners}
+ * @ignore
+ */
+function _transformAPIAuthorizingIntegrationOwners(authorizingIntegrationOwners) {
+  return Object.fromEntries(Object.entries(authorizingIntegrationOwners).map(([key, owner]) => [parseInt(key), owner]));
```

Comment:
This is useless, isn't it? You're just ending up cloning the object received 👀 (the key will automatically be turned into a string anyways)